package.json

```
@@ -16,7 +16,11 @@
     "package": "vsce package -o wit-idl.vsix",
     "update-snapshot": "vscode-tmgrammar-snap --updateSnapshot --scope 'source.wit' 'tests/grammar/*'",
     "test-grammar": "vscode-tmgrammar-test -c --grammar syntaxes/wit.tmLanguage.json 'tests/grammar/*'",
-    "test": "npm run test-grammar"
+    "test": "npm run test-grammar",
+    "vscode:prepublish": "npm run compile",
+    "compile": "tsc -p ./",
+    "watch": "tsc -watch -p ./",
+    "lint": "eslint src --ext ts"
   },
   "engines": {
     "vscode": "^1.65.0"
```

Change needed so that `activationEvents` property is added automatically:
```suggestion
    "vscode": "^1.75.0"
```